README.md

```diff
@@ -1,12 +1,76 @@
 # hello-world-fwless
-Simple java API rest without framework
+Simple java rest API without framework.
 
+This branch aims to showcase how to use [Dekorate](https://github.com/dekorateio/dekorate) to deploy this microservice on a Kubernetes cluster.
 
-This branch aims to showcase how to use Dekorate to deploy this microservice on Openshift.
+## Set up
+This project uses [kind](https://kind.sigs.k8s.io/) for running a Kubernetes cluster locally. You can install it following the instructions [here](https://kind.sigs.k8s.io/docs/user/quick-start/#installation).
+Once installed, we need to set it up in order to use a local docker registry and install the ingress controller to make Ingress resources work. So follow the next steps:
```

Comment:
Once installed, we need to set it up in order to use a local docker registry and install the ingress controller to make Ingress resources work.

-->

Once installed, we need to set it up in order to :
- Use a local docker registry needed to provide the image of the microservice when the pod is created
- Install an Ingress controller managing the access from the host to the cluster of the services exposed